apps/nextjs-website/src/helpers/metadata.helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { translations } from '@/_contents/translations';
+import { ResolvedMetadata } from 'next';
+import { AbsoluteTemplateString } from 'next/dist/lib/metadata/types/metadata-types';
+import { Twitter } from 'next/dist/lib/metadata/types/twitter-types';
+
+export const getTitleFromMarkdown = (markdown: string): string => {
+  const lines = markdown.split('\n').filter((line) => line !== '');
+  const titles = lines.filter((line) => line.startsWith('#'));
+  const title = titles[0]?.replace('# ', '').trim() ?? '';
+  return title;
+};
+
+export const getTwitterMetadata = (title: string): Twitter => ({
+  title,
+  card: 'summary',
+  site: '@pagopa',
+  creator: '@pagopa',
+});
+
+export const getPreviousTitle = async (
```

Comment:
why the function should be `async`?

btw, returning a `Promise` you are having issues with the generated `meta` tags value: if you run the application and take a look at the values of the tag you will see something like `content="[object Promise] - L'App IO"`